### ISSUE TITLE
Sprint - Jan 8 to Jan 19

### ISSUE THREAD

**Issue by mariusandra** [PostHog/posthog#19572]
# Global Sprint Planning

## 3 things that might take us down

1. James G isn't available when ClickHouse is broken
2.
3.

# Team sprint planning

For your team sprint planning copy this template into a comment below for each team.

```
# Team ___

**Support hero:** ___

## Retro

<!-- Grab the high and low priority items from last time and add whether that item was completed or not -->

- 

## Hang over items from previous sprint

<!-- For each item, decide to re-prioritise (and add below) or deprioritise -->

- Item 1. prioritised/deprioritise

## OKR

1. OKR, status (red/yellow/green) and action points if yellow/red


### High priority

-

### Low priority / side quests

-

```

**Comment by mariusandra** [comment]
# Team ~Query Engine~ Product Analytics

**Support hero:** 
- TBD

## Retro

- Christmas week: keeping the lights on.
   - Christmas passed, lights are still on.

## OKR Q1 2024

1. HogQL & Data Exploration (Julian, Marius)
   - Convert ALL our insights to use HogQL as their base.
   - Improve the type system and get rid of `assumeNotNull`.
   - Write great docs for HogQL and data exploration nodes.
   - Proactively monitor query performance.

2. Product Analytics frontend (Thomas, Michael)
   - Clean things up now that we have PostHog 3000 and HogQL everywhere.
   - Work on the universal "Explore" view. 

3. BI (Tom)
   - Support non-event data sources on HogQL insights.
   - Build the new querying experience, and/or integrate it into the "Explore" view.

Action plan: 
- Come January, we'll take [this list](https://github.com/PostHog/meta/issues/130), split the checkboxes between the team, and keep going till it's done. ETA: end of January.
- In February (leap year, woohoo), we'll make Product Analytics ~great~ spark joy again: insights UI cleanup, unified data exploration experience, adding missing analysis features (lifecycle breakdowns, etc), and in parallel work on the BI and the HogQL editing experience. What _exactly_ we'll do and in what order is still TBD.
- In March we'll keep going. The goal is to have an amazing BI / Product Analytics / Insights / Explore / HogQL experience by the end of it.
## Next sprint

### High priority

- HogQL insights rewrite.

### Low priority / side quests

- Everything else.

**Comment by pauldambra** [comment]
# Team Replay

**Support hero:**  @pauldambra 

@daibhin off for most of sprint
@pauldambra off on the 8th Jan

## Retro
- xmas was lovely @pauldambra 
- frustrating to not get feedback from customers yet, hopefully due to the holidays 🤞 @marandaneto 

## Hang over items from previous sprint

✅ (Wrap up comments work) - ben releasing it today, needs some follow-up
✅ Investigate [canvas replay](https://github.com/PostHog/posthog/issues/14555) @daibhin 
    - we have a plan for how to implement this after David's marathon
❌ Get a user actively testing the SDK @marandaneto 
holidays hopefully
❌ Mobile [dashboard templates](https://github.com/PostHog/posthog/issues/19341) @marandaneto @pauldambra 
   - have a draft of this... will aim to complete this week but it might carry over if other higher priority work interrupts
♻️ [Mutations](https://github.com/PostHog/posthog-android/pull/72) in mobile replay @marandaneto @pauldambra 
    - backend and SDK changes are in, we're gonna test this week to see what the gap between theory and implementation is
✅ Get feedback on the network payload stuff from users @pauldambra 
♻️ [Update Flutter SDK](https://github.com/PostHog/posthog-flutter/pull/60) to support feature success things @marandaneto 
    - in progress, needs more post-xmas work, actively testing with an engaged user, aiming to complete this sprint
❌ Churn investigation work @daibhin 
    - nothing happened on this... new plans being released today

## OKR
* 🟡📱 happy android replay users
* 🌜 people love replay even more
* 🌜first class mobile support
  
### High priority

* make sure the plans and controls change lands well with users @pauldambra 
* 2 users using and have given us feedback on Android replay @marandaneto 
* mobile dashboard template released @pauldambra 

### Side quests

* network payload capture released for all @pauldambra 
* BETA canvas recording for replay - @daibhin has 3 days after leave... 
* POC of session replay with jetpack compose @marandaneto 
* @daibhin take over this PR to add resizeable inspector tab for the player https://github.com/PostHog/posthog/pull/18378

**Comment by EDsCODE** [comment]
# Team Data Warehouse

### OKR Q1 2024

### Objective

**Get EU moved over to new cluster**

* **Key Results:**
    * Terraform, ansible, data migrated over, backups, housewatch
    * Deprecate old cluster

**ByConity evaluation**

* **Key Results:**
    * Load up with all clickhouse data and benchmark

**5 reference customers with qualities:**

* **Key Results:**
    * Paying a reasonable amount
    * Tell us they’re delighted
    * Not a big edge case
    * Using our in house data import pipeline
    * Weekly queries on a data warehouse connected table
    * Saving queried tables to insights?

**Data Warehouse queries are compatible with insights hogQL**

 

### Retro

**Product**
- [x] billing/row limits (otherwise large volume customer jobs can run forever)
- [ ] add all schemas as options for stripe connector
- [x] release to more teams
- [x] hubspot integration


**Infra**
- [ ] https://github.com/PostHog/posthog/issues/19420
- [ ] https://github.com/PostHog/posthog/issues/19421
- [ ] https://github.com/PostHog/posthog/issues/19422
- [ ] https://github.com/PostHog/posthog/issues/19423
- [ ] https://github.com/PostHog/posthog/issues/19456
- [ ] https://github.com/PostHog/posthog/issues/19457
- [ ] Add projection for historical exports

### High priority

**Product**
- [ ] release to more users. Clean up intro panels so first layer of data warehouse external sources can be fully released
- [ ] onboard more users to data connectors + release hubspot integration to interested users
- [ ] continue working through conversion from billing queries on metabase to within PostHog


**Infra** [Board](https://github.com/orgs/PostHog/projects/92/views/1)

**Comment by neilkakkar** [comment]
# Team Feature Success

**Support hero:** @liyiy (surveys) + @jurajmajerik  
Days off:
Juraj: 0 days
Neil: 2 days
Li: 0 days

## Retro

- Fix mismatch between experiment person modal & user count in funnel - @jurajmajerik -> sprint begins tomorrow, ends day after, so we'll find out 😅 
- SDK updates for relative date targeting & targeting by distinct id - @neilkakkar -> was unexpectedly sick , 4/5 libs down. 1 to rollover.
- e2e tests for surveys & timer for delayed surveys - @liyiy -> e2e tests in progress, timer will roll over.


## Hang over items from previous sprint

-  timer for delayed surveys to roll over indefinitely

---

## OKRs

1. Doubling down on surveys
2. Doubling down on experiments

### High priority

- Release scheduled flag changes, fix some experiment bugs to get comfortable with the codebase to prep for Q1 OKRs ( #18104 , #18958 ) - @jurajmajerik 
- Release relative date targeting & targeting by distinct id - @neilkakkar 
    - and setup temporal for feature success tasks
- Convert surveys to react [or whatever makes most sense here], and use it for previews in posthog app - @liyiy 

### Low priority / side quests

**Comment by davemurphysf** [comment]
# Previous sprint

1. Webhooks -> running for team2 prod

- ✅ Support for non-tx and tx modes (we may commit to one or the other later).
- ✅ Hook-consumer metrics.
- 🟡 Finishing up hook-janitor work for cleaning-up jobs and emitting metrics.
         Testing setup:
         - ✅ Dockerfile
         - ☑️ Provision PG (to finish this week)
         - ☑️ Deployment chart (to finish this week)

2. PoE:
- ☑️ Ramp up new override path with more teams
- 🟡 We have a plan for the backfill
                Turned over a rock on splits…

3. Capture-rs:
- ✅ Prod-us rollout for the 20 remaining teams
- ✅ Prod-eu rollout (first all teams but one, then that last one)
  - => Enabled for everyone except OAK (bad reverse proxy config) and MX (awaiting info on proxy), considering done and will follow-up for these two orgs after they reply.

# Next sprint

Luigi: Tomás then Tiina

Wrap up Q4 work before offsite in Jan
  	- Finish webhooks
    	- Delivery system (Brett, Tomás)
	- Deployment and monitoring ( , Xavier)
	- Make plugin-server delegate to hooks (Xavier)
        - Pipeline UI (Tiina)
               - data pipelines unsubscribe option
- new UI released
  	- Finish POE (Ted)
		- Write RFC for persons splits
		- Move forward with backfill plan
		- Continuing write ramp up

**Comment by raquelmsmith** [comment]
# Team Growth

## Retro
<details>
<summary>Retro items</summary>
    - [x] Look into emails that happen after initial product analytics and session replay instrumentation, come up with an email to try to get people doing one of the activation events
      - run as experiment
      - [x] @xrdt for product analytics
      - [x] @raquelmsmith for session replay
      - [ ] Running three experimental campaigns: 
        - [x] [You've got replays](https://fly.customer.io/workspaces/127208/journeys/campaigns)
        - [x] Product analytics [ingestion](https://fly.customer.io/workspaces/127208/journeys/campaigns)
        - [x] Product analytics [insight](https://fly.customer.io/workspaces/127208/journeys/campaigns)
</details>


## OKR
- Double activation rates for product analytics and session replay
  - Polish onboarding flow with Cory
  - Run more smaller experiments
  - Do an average of 2 customer interviews per week
- Launch pricing for Data warehouse, queries, CDP


## This sprint
### High priority
  - [ ] Full testing of the onboarding flow @raquelmsmith 
  - [ ] New session replay plan
    - https://github.com/PostHog/product-internal/pull/547
    - New plan and price
    - Feature gating based on available_product_features (right now it's behind a flag)
  - [ ] Data pipelines pricing https://github.com/PostHog/billing/issues/417
    - https://github.com/PostHog/product-internal/pull/533/files
    - [ ] Unsubscribe modal https://github.com/PostHog/posthog/pull/19334
    - [ ] Create product in billing
    - [ ] Add feature gates
    - [ ] Organize launch plan


### Low priority / side quests
## Growth review items
- [ ] Add metadata to startup plan customers in stripe @raquelmsmith 
- [ ] ctas on billing for customers over 1k/mo @xrdt https://github.com/PostHog/posthog/pull/19508
  - Bundling discounts
  - Team training
  - Roadmap chit chat 
- [ ] We shouldn’t show pricing above $20k/yr on the pricing pages so we don’t “shoot ourselves in the foot” for larger customers who will be paying above that
  - Could be an experiment, does it put people off from signing up because it isn’t as transparent, or feels enterprise-y.
